code/modules/antagonists/wizard/wizard.dm

```diff
@@ -332,6 +332,11 @@
 
 	return parts.Join("<br>")
 
+/datum/antagonist/wizard/check_mastery()
+	if(length(owner.spell_list))
+		return
+	owner.current.client.give_award(/datum/award/achievement/antagmastery/wizard, owner.current)
```

Comment:
This one also seems a bit boring, wizard is a murderbone antag so you are kind of free to put anything on these

Reply:
This one is designed to encourage off meta builds, as all the meta builds use spells, whereas this forces you to use minions, artifacts, wands, staves, etc.